arches/app/templates/views/components/photo-workbench.htm

```diff
@@ -110,17 +110,17 @@ <h4>
                        </button>
                        <br>
                        <div style="padding: 10px">
-                            <p>
-                                <span data-bind="text: $root.translations.addingPhotosOptional"></span>
-                            </p>
-                            <p>
-                                <span data-bind="text: $root.translations.maxPhotoSizeWarning(maxFilesize())"></span>
-                            </p>
-                        </div>
+                        <p>
+                            <span data-bind="text: $root.translations.addingPhotosOptional"></span>
+                        </p>
+                        <p>
+                            <span data-bind="text: $root.translations.maxPhotoSizeWarning + ' ' + maxFilesize() + ' MB.'"></span>
+                        </p>
+                    </div>
 
                    </div>
                    <div class="file-upload-footer">
-                        <span data-bind="text: $root.translations.validImageFormats"></span>
+                        <span data-bind="text: $root.translations.validImageFormats  + ' ' + acceptedFiles()"></span>
```

Comment:
I think in RTL the variable will get appended to the wrong side of the sentence.  Could it be passed in a function like `maxPhotoSizeWarning`. That would give translators control of the formatting.